CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# CONTRIBUTING
+
+Contributions to the XRPL-Standards repo are free, open source, and follow the following document publishing process.
+
+Any XRPL-Standards document can be referred to interchangeably as an "XLS" (XRPL Standard), "RFC" (Request for Comments), or "document".
+
+## Summary
+
+Copyright on all content is subject to the terms of the [MIT License](LICENSE).
+
+All contributors grant everyone:
+
+Copyright – a royalty-free license to use the copyrights for their contributions.
+Patent – a commitment to license on a royalty-free basis their essential patent claims reading on their contributions.
+
+## Background
+
+The work of the XRP Ledger community is being done under the framework of a W3C Community Group and is therefore guided by the [processes](https://www.w3.org/community/about/agreements/) laid out by the W3C for these groups.
```

Comment:
I think we should delete this section -- there's no W3C association that I'm aware of.